src/index.html

```diff
@@ -1,13 +1,37 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Serif:wght@500&family=Roboto:wght@300;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Product cards</title>
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <div class="container-image"><img class="image-imac" src="./images/imac.jpeg" alt="IMac"></div>
+      <div class="container-info">
+        <div class="container-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
```

Comment:
```suggestion
        <div class="container-name">
          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
        </div>
```
И так везде лучше делать (кроме очень коротеньких блоков)